static/js/reducers/comments.js

```diff
@@ -1,18 +1,19 @@
 // @flow
-import { GET, INITIAL_STATE } from "redux-hammock/constants"
+import { GET, POST, INITIAL_STATE } from "redux-hammock/constants"
 
-import { makeCommentTree } from "../factories/comments"
+import * as api from "../lib/api"
 
 import type { Comment, Post } from "../flow/discussionTypes.js"
 
 export const commentsEndpoint = {
   name:              "comments",
-  verbs:             [GET],
-  getFunc:           async (post: Post) => makeCommentTree(post),
+  verbs:             [GET, POST],
+  getFunc:           (post: Post) => api.getComments(post.id),
   initialState:      { ...INITIAL_STATE, data: new Map() },
   getSuccessHandler: (comments: Array<Comment>, data: Map<string, Array<Comment>>) => {
     let update = new Map(data)
     update.set(comments[0].post_id, comments)
     return update
-  }
+  },
+  postFunc: (post: Post, comment: string, commentId: ?string) => api.createComment(post.id, comment, commentId)
```

Comment:
Figured out why I was getting the 403, this code accepts a `Post` object, but the test code in your PR description says to pass just the id. Code works with the `Post`, but just want to confirm which one is the preferred api here. This code is also subject to the other error I posted.